tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -313,6 +314,39 @@ var _ = Describe("Verify Create", Ordered, func() {
 				g.Expect(res).Should(ContainSubstring("local-path-test"))
 			}, "180s", "2s").Should(Succeed())
 		})
+
+		It("Verifies Certificate Rotation", func() {
```

Comment:
@rancher-max  @est-suse   also what you guys think to separete the code in block context for be more clean? some like this : 

```
const x
const y
var finalResult string
var finalErr error

errStop := e2e.StopCluster(serverNodeNames)
Expect(errStop).NotTo(HaveOccurred(), "Server not stop correctly")

errRotate := e2e.RotateCertificate(serverNodeNames)
Expect(errRotate).NotTo(HaveOccurred(), "Certificate not rotate correctly")

errStart := e2e.StartCluster(serverNodeNames)
Expect(errStart).NotTo(HaveOccurred(), "Server not start correctly")

for _, nodeName := range serverNodeNames {
    grCert, errGrep := e2e.RunCmdOnNode(grepCert, nodeName)
    Expect(errGrep).NotTo(HaveOccurred(), "Certificate not created correctly")
...
```